### PR TITLE
Feature documentation

### PR DESCRIPTION
**Pull request by bikegeek**
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
Build documentation without errors or warnings
Visually inspected content
- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ ] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes or No]**

- [ ] Do these changes include sufficient testing updates? **[Yes or No]**

- [ ] Will this PR result in changes to the test suite? **[Yes or No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR definition above.
- [ ] Ensure the PR title matches the feature or bugfix branch name.
- [ ] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [ ] After submitting the PR, select **Development** with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

**Review by jvigh — CHANGES_REQUESTED** [review]
@bikegeek I have reviewed the documentation and made some suggested changes via a commit.  I added a bit of detail to the introduction section to provide more context on this specific TC-Diag plotting capability.  I also made some minor wordsmithing changes throughout and moved the example to the end, as I thought it might make more sense to describing the procedure for running and then show them the expected result.  Feel free to delete any of these changes that is not useful, or if it contravenes some documentation standards or styles that I don't know about.

**Comment by jvigh** [comment]
@bikegeek   Question: Since this capability is based off of MetPy's Skew-T functionality, should it be credited somewhere?

Also, if references are permitted, the following could be useful for readers who are unfamiliar with Skew-T:
https://en.wikipedia.org/wiki/Skew-T_log-P_diagram
https://www.e-education.psu.edu/meteo300/node/638

**Comment by bikegeek** [comment]
Attribution should be given in publications, so if the project is going to generate a publication, we should place the attribution in the reference section (https://unidata.github.io/MetPy/latest/userguide/citing.html).  We are using Metpy software, which has BSD license.

File: docs/Users_Guide/skew_t.rst [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+   Output files are named with the same filename as the input file, with the *.png* extension replacing the
+   *.dat* extension of the data file. The
+   plot title is generated from the name of the input file and the corresponding sounding hour.
+
+Example output plot
+___________________
+
+The **skew_t.yaml** configuration file in combination with the
+Python script and sample data, generates the following skew-T plot:
+.. image:: figure/ssh052023_avno_doper_2023010100_diag_0_hr.png
```
Comment (jvigh) [inline docs/Users_Guide/skew_t.rst:158]: The image does not seem to render in RTD: 

image

Is there a syntax issue with how it is being included?

File: docs/Users_Guide/skew_t.rst [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+*/usr/path/to/METplotpy*  if the source code was cloned or forked from the Github repository
+
+or
+
+*/usr/path/to/METplotpy-x.y.z*  if the source code was downloaded as a zip or gzip'd tar file from the Release link of
+the Github repository.  The *x.y.z* is the release number.
+
+
+For background information on the CIRA data, please refer to:
```
Comment (jvigh) [inline docs/Users_Guide/skew_t.rst]: Recommend changing "data" to "TC model diagnostics".

File: docs/Users_Guide/skew_t.rst [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+McNoldy, B. D., K. D. Musgrave, and M. DeMaria, 2012: Diagnostics and verification of the tropical cyclone environment in regional models. AMS 30th Conference on Hurricanes and Tropical Meteorology, April 2012, Ponte Vedra Beach, Florida.
+
+https://bmcnoldy.rsmas.miami.edu/papers/MMD_30HURR.pdf
```
Comment (jvigh) [inline docs/Users_Guide/skew_t.rst:42]: Question: I thought Kate was going to give you a link to where users could obtain the real-time data. Has she given you this yet?

Comment (bikegeek) [reply docs/Users_Guide/skew_t.rst:42]: Kate was going to provide a link to the data but CIRA is migrating to something else and no longer using ftp.

File: docs/Users_Guide/skew_t.rst (hunk 2 above)
Comment (bikegeek) [reply docs/Users_Guide/skew_t.rst]: Changed CIRA data to CIRA TC model diagnostics.

File: docs/Users_Guide/skew_t.rst (hunk 1 above)
Comment (bikegeek) [reply docs/Users_Guide/skew_t.rst:158]: needed to add another carriage return in between the previous line and the image syntax.  Now the image is rendering.

**Review by jvigh — APPROVED** [review]
I see your changes. Everything looks good. I approve this pull request. Thanks!

**Comment by bikegeek** [comment]
Thanks, this should be the last PR for the METplus analysis tools for you
(for the coordinated release).

---------------
Minna Win
Pronouns: she/her
National Center for Atmospheric Research
Research Applications Lab
Developmental Testbed Center
Phone: 303-497-8423
Fax:   303-497-8401
---------------

**Pull request merged**